src/Files.App/MainWindow.xaml.cs

```diff
@@ -45,7 +46,7 @@ public MainWindow()
 		private void EnsureEarlyWindow()
 		{
 			// Set title
-			AppWindow.Title = "Files";
+			AppWindow.Title = SystemInformation.Instance.ApplicationName;
```

Comment:
```suggestion
			AppWindow.Title = "Files";
```

Reply:
Why? I do want it to distinguish between Dev, Preview and the release version.

Reply:
The icon helps with that